packages/clerk-js/src/ui/customizables/parseVariables.ts

```diff
@@ -19,6 +20,7 @@ export const createColorScales = (theme: Theme) => {
     colorInputText: toHSLA(variables.colorInputText),
     colorBackground: toHSLA(variables.colorBackground),
     colorInputBackground: toHSLA(variables.colorInputBackground),
+    colorAvatarGradient: (variables.colorAvatarGradient as string[])?.map(color => toHSLA(color)),
```

Comment:
```suggestion
    colorAvatarGradient: variables.colorAvatarGradient ? variables.colorAvatarGradient.map(toHSLA) : undefined,
```

Note 1: I removed the ternary to make the `undefined` value more visible. This is a rather complex part of the codebase so let's prefer clarity over shortcuts 

Note 2: If `colorAvatarGradient` gets `undefined` as a value, `parseAppearance` will use the default value from the internal theme as a fallback in the next step